SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Utils/IDeviceInformationHelper.cs

```diff
@@ -11,9 +11,15 @@ namespace Microsoft.AppCenter.Utils
     public interface IDeviceInformationHelper
     {
         /// <summary>
-        /// Gets the device information.
+        /// Gets the device information asynchronously.
         /// </summary>
         /// <returns>Device object with fields populated appropriately</returns>
         Task<Ingestion.Models.Device> GetDeviceInformationAsync();
+
+        /// <summary>
+        /// Gets the device information without blocking. This might be missing certain properties.
+        /// </summary>
+        /// <returns>Device object with fields populated appropriately</returns>
+        Ingestion.Models.Device GetDeviceInformation();
```

Comment:
This method does not need to be in the interface. The code in this branch always access it without the interface and thus resharper complains.

Either use interface or remove the method from the interface to resolve the warning.

Reply:
The method is accessed on the interface in line 69 of ErrorLogHelper.cs. Maybe Resharper is confused since it is across multiple projects?